open-metadata-implementation/view-services/tex-view/tex-view-server/src/main/java/org/odpi/openmetadata/viewservices/tex/admin/TexViewAdmin.java

```diff
@@ -103,7 +141,7 @@ public void initialize(String            serverName,
                                   error);
 
             super.throwUnexpectedInitializationException(actionDescription,
-                                                         ViewServiceDescription.TYPE_EXPLORER.getViewServiceFullName(),
+                                                         ViewServiceDescription.DINO.getViewServiceFullName(),
```

Comment:
should this be Type explorer?

Reply:
Yup - good spot - thanks. I had even search for all occurrences of 'dino' but had match case enabled - d'oh!

Reply:
Found another (mixed case) example the same way - thanks again!